vignettes/createReposHtml.Rmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "HOWTO generate repository HTML"
+author: "S. Falcon"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{biocViews-CreateRepositoryHTML}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+    number_sections: yes
+    toc: yes
+    toc_depth: 4
+---
+
+# Overview
+
+This document assumes you have a collection of R packages on local disk that you
+would like to prepare for publishing to the web. The end result we are going for 
+is:
+
+1. Packages organized per CRAN-style repository standard.
+2. PACKAGES files created for install.packages access.
+3. VIEWS file created for generating biocViews.
+4. A vignette directory created containing the extracted vignette pdf files from 
+   each source package in the repository.
+5. An html directory created containing html descriptions of each package with 
+   links for downloading available artifacts.
+6. A simple alphabetical listing index.html file.
+
+# CRAN-style Layout
+
+Establish a top-level directory for the repository, we will refer to this 
+directory as reposRoot. Place your packages as follows:
+
+**src/contrib** Contains all source packages (*.tar.gz).
+
+**bin/windows/contrib/x.y** Contains all win.binary packages (*.zip). Where x.y 
+  is the major.minor version number of R.
+
+**bin/macosx/contrib/x.y** Contains the mac.binary (High Sierra) (*.tgz) 
+  packages.
+
+You will need the following parameters:
+
+```{r echo=TRUE, eval=FALSE}
```

Comment:
No need to repeat `echo=TRUE` when this is the default: 

```r
knitr::opts_chunk$get("echo")
[1] TRUE
```

Reply:
(Note to self) Completed.